backend/canisters/user/api/can.did

```diff
@@ -565,6 +565,32 @@ type UnArchiveChatResponse =
         ChatNotFound;
     };
 
+type AddRemoveFavouriteChatsArgs =
+    record {
+        to_add: vec Chat;
+        to_remove: vec Chat;
+    };
+
+type AddRemoveFavouriteChatsResponse =
+    variant {
+        Success;
+        UserSuspended;
+    };
+
+type ArchiveUnarchiveChatsArgs =
+    record {
+        to_archive: vec Chat;
+        to_unarchive: vec Chat;
+    };
+
+type ArchiveUnarchiveChatsResponse =
+    variant {
+        Success;
+        PartialSuccess: record { chats_not_found: vec Chat; };
+        Failure: record { chats_not_found: vec Chat; };
```

Comment:
Failure means everything failed, right?
So surely it doesn't need the list of chats?